DESCRIPTION

```diff
@@ -29,11 +30,11 @@ Suggests:
     vdiffr,
     knitr,
     rmarkdown,
-    survminer,
     forcats,
     glue,
-    here,
-    stringr
+    stringr,
```

Comment:
if we can avoid this one, please do. It does not have added value as all string manipulations are straightforward with base R

Reply:
Removed both from the release branch DESCRIPTION file